tools/qc/fastq_pair/.shed.yml

```diff
@@ -0,0 +1,10 @@
+name: fastq_pair
+owner: ebi-gxa
+description: "Paired-end fastq pairer"
+long_description: "Match up paired end fastq files quickly and efficiently. Rewrite paired-end fastq files to make sure that all reads have a mate and to separate out singletons."
+homepage_url: https://github.com/linsalrob/fastq-pair
+remote_repository_url: https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/tree/develop/tools/qc/fastq_pair
+type: unrestricted
+categories:
+- Transcriptomics
+- RNA
```

Comment:
I would suggest to include here 
- Fastq Manipulation

Reply:
Thanks, done in 098b9e8a14e6c2382b1cb6874a19efc0318651f4. I also removed the other categoriees; your suggestion is more suited for the tool.

Reply:
Checked the toolshed, `Fastq Manipulation` is there.